src/clients/api/queries/getIsAddressAuthorized/useGetIsAddressAuthorized.ts

```diff
@@ -19,7 +18,10 @@ const useGetIsAddressAuthorized = (accountAddress: string, options?: Options) =>
     FunctionKey.GET_IS_ADDRESS_AUTHORIZED,
     () => getIsAddressAuthorized({ accountAddress }),
     {
-      refetchInterval: DEFAULT_REFETCH_INTERVAL_MS,
+      refetchOnMount: false,
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+      staleTime: Infinity,
```

Comment:
I'd say this is too permissive. It can be the case that DEFAULT_REFETCH_INTERVAL_MS refetches too frequently and we don't want it, but I think we need to check it from time to time.

Reply:
Agreed, Jesús also thinks the same. Perhaps once every hour then?